app/src/main/java/ch/epfl/sweng/swenggolf/database/FireDatabase.java

```diff
@@ -82,27 +85,30 @@ public void onComplete(DatabaseError databaseError,
     }
 
     @Override
-    public void write(@NonNull String path, @NonNull String id, @NonNull Object object) {
+    public void write(@NonNull String path, @NonNull String id, @NonNull Object object){
         database.getReference(path).child(id).setValue(object);
     }
 
     @Override
     public void write(@NonNull String path, @NonNull String id, @NonNull Object object,
-                      @NonNull final CompletionListener listener) {
+    @NonNull final CompletionListener listener){
```

Comment:
Did you reformat the code propoerly? The spacing seems odd. Maybe you could reformat the whole java folder to be sure that everything looks good.

Reply:
I think I did but I will redo it anyway to make sure, good catch